### DISCUSSION TITLE
Notifications: Using the JavaScript objects not working

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#11574] [Help]
### Package

Notifications

### Package Version

v3.2.37

### How can we help you?

I am trying to use Notifications in [Using the JavaScript objects](https://filamentphp.com/docs/3.x/notifications/sending-notifications#using-the-javascript-objects) but this problem occurs.
$${\color {red}Uncaught \space SyntaxError: \space The \space requested \space module \space '/vendor/filament/notifications/dist/index.js' \space does \space not \space provide \space an \space export \space named \space 'Notification'}$$

**Comment by leandrocfe** [comment]
Are you using Panel builder or Notifications package only?

**Reply by ArtMin96** [reply]
Notification package only

**Comment by micraux** [comment]
Hello @ArtMin96,

Haven't you got a typo in your file name?

Maybe `index.js` instead of `indez.js`?

**Reply by ArtMin96** [reply]
It looks like `indez.js` because of the font, but actually, it is index

**Reply by micraux** [reply]
Oops, sorry for that one!
